src/test/java/org/cactoos/io/ZipTest.java

```diff
@@ -67,7 +67,7 @@ public void zip() throws Exception {
                 "Can't list files in a directory represented by a path",
                 cnt,
                 // @checkstyle MagicNumber (1 line)
-                Matchers.equalTo(4)
+                Matchers.equalTo(1)
```

Comment:
@fanifieiev Please replace this matcher with a direct comparison to an iterable of path as strings. (instead of counting them, let's compare them precisely).